src/matchmaker/competition.js

```diff
@@ -27,7 +27,10 @@ function drawFirstRound({ players }) {
   // 3. create matches
   const newMatches = createMatches(pairings);
 
-  // 4. update players
+  //4. play matches
```

Comment:
I would remove step 4 and 5 out of the drawing function.

For me the darwing function should just create the newMatches and return them.

The playMatches and updatePlayers/Matches function should be somewhere else because this is in my opinion not part of the drawing itself

Reply:
Yeah, let's also talk about this on wednesday 👍